public/css/app.css

```diff
@@ -11987,79 +11987,3 @@ a.fc-more:hover {
   min-width: 1.25em;
 }
 
-/* Links */
```

Comment:
I believe we should revert those. The menu is not responsive it closes on click

Reply:
I believe I can revert the changes on top of your commit, this comment is for me I can checkout your branch, take the commit and fix what needs to be fixed. This comment is for me